contact[1].php

```diff
@@ -10,21 +20,40 @@
 		<meta name="author" content="Johno Devine">
 		<meta name="viewport" content="width=device-width, initial-scale=1">
 		<link rel="stylesheet" href="stylesheet.css"/>
+		<link rel="icon" type="image/x-icon" href="images/icon.ico">
 	</head>
 	<body>
+		<!-- background-->
 		<div class="leftbackground"></div>
 		<div class="rightbackground"></div>
+		<!-- checks if the user logged in is an admin or not-->
+		<?php
+		if(!isset($_SESSION['Form'])){
+			 ?>
+			<button class="btn" onclick="window.location.href='login.php';"> <img src="images/profile.png" class="profile_img"/></button>
+			<?php
+		}
```

Comment:
php that detrieme what to show the user if they are an Admin (Graeme) or a normal user